### PR TITLE
replace all instance of wiki.alephium.org with docs.alephium.org

### PR DESCRIPTION
**Pull request by Sakrecoer**
as the title suggests :) please have a look @nop33 or @mvaivre

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -63,11 +63,11 @@ Our VM and language are really dev-friendly and if you're familiar with Solidity
 
 ### Where is the roadmap?
 
-You will find it on the Wiki: https://wiki.alephium.org/#roadmap and on the website: https://alephium.org/#what's_next. You can also follow our weekly development updates on Discord, Twitter or Reddit.
+You will find it on the Wiki: https://docs.alephium.org/#roadmap and on the website: https://alephium.org/#what's_next. You can also follow our weekly development updates on Discord, Twitter or Reddit.
 
 ### Where can i query an API?
```
Comment (mvaivre) [inline docs/FAQ.md]: Not linked to this PR, but 
can I query... "i" should be uppercase 🙃

File: i18n/fr/docusaurus-plugin-content-docs/current/5min-overview.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ Rencontrez le reste de l'équipe sur notre site web ou sur Linkedin : <https://w
 
 ### Chattez en votre langue
```
Comment (mvaivre) [inline i18n/fr/docusaurus-plugin-content-docs/current/5min-overview.md]: Again not linked to this PR (sorry, I just highlight the little things I come across, continuous improvement ;)) : "Chattez *dans* votre langue".

**Review by mvaivre — APPROVED** [review]
👍

File: docs/FAQ.md (hunk 1 above)
Comment (Sakrecoer) [reply docs/FAQ.md]: merci! fixed in https://github.com/alephium/wiki/pull/118/commits/cc648b062452f7d9350461a5addb712474bb4446

File: i18n/fr/docusaurus-plugin-content-docs/current/5min-overview.md (hunk 2 above)
Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/5min-overview.md]: merci! fixed in https://github.com/alephium/wiki/pull/118/commits/cc648b062452f7d9350461a5addb712474bb4446

**Pull request merged**